tests/shape_test.py

```diff
@@ -8,6 +8,12 @@
 from .common import assert_export
 
 
+def test_broadcast():
+    x = sc.array(dims=['x'], values=np.arange(6.0))
+    assert_export(sc.broadcast, x=x, sizes={'x': 6, 'y': 3})
+    assert_export(sc.broadcast, x=x, dims=['x', 'y'], shape=[6, 3])
```

Comment:
There is some logic in the wrapper for handling the different args. Can you extend the checks to make sure that the output has the correct shape? (Also that there is an exception when both sizes and dims / shape are passed)

Reply:
Now you could remove the `assert_export` calls. But that is just a minor thing